pipelines/folioRancher/folioQualityGates/Jenkinsfile

```diff
@@ -27,20 +29,36 @@ pipeline {
             }
         }
 
-        stage('Check Changes') {
-            steps {
-                script {
-                    if (params.SKIP_CHANGES == true) {
-                        echo "Skipping changes For the SNAPSHOT BRANCH and moving to Parrarel Run ."
-                    } else {
-                        checkout scm
-                        echo "Checking Changes for the SNAPSHOT BRANCH..and moving to Parrarel Run ."
-                    }
+stage('Check Changes') {
+    steps {
+        script {
+           if (params.SKIP_CHANGES) {
```

Comment:
Maybe better to name this param like 'SKIP_CHECK'?

Reply:
Adjusted, in new commit will be included